src/App.css

```diff
@@ -4,7 +4,422 @@
   box-sizing: border-box;
 }
 
+header {
+  width: 100%;
+  height: auto;
+  z-index: 1;
```

Comment:
- [ ] Please, consider fixing some points in styling to cover the original design:

1. Try to move right on the same line with the books list the title "Bookstore CMS" with links, as shown on the screenshot
2. Please, fix the title of the links with uppercase.
3. Try to move left the account icon, as it is in the original design.

<img width="1348" alt="Screen Shot 2022-04-01 at 20 33 48" src="https://user-images.githubusercontent.com/49064106/161311073-6d093a59-79ea-4799-8e67-10c2f1cd82d4.png">

- [ ] Please, try to fix the alignment with the same line of books list, as shown on the screenshot 👍 
<img width="1365" alt="Screen Shot 2022-04-01 at 21 13 24" src="https://user-images.githubusercontent.com/49064106/161311632-a3578fe5-923a-4f12-8700-41ecdbc315e9.png">